src/highdicom/seg/sop.py

```diff
@@ -2174,7 +2174,7 @@ def _get_ref_instance_uids(self) -> List[Tuple[str, str, str]]:
                 f'duplicated referenced SOP Instance UID items: {display_str}.'
             )
 
-        return list(unique_instance_data)
+        return unique_instance_data.copy()
```

Comment:
I agree, I think this `list()` call is vestigial, let's remove outright
```suggestion
        return unique_instance_data
```